h2o-persist-hdfs/build.gradle

```diff
@@ -1,13 +1,41 @@
 
 description = "H2O Persist HDFS"
 
+configurations {
+    testCompile.exclude module: 'servlet-api'
+}
+
 dependencies {
     compile project(":h2o-core")
-    compile('net.java.dev.jets3t:jets3t:0.6.1')
+    compile('net.java.dev.jets3t:jets3t:0.9.0')
```

Comment:
Hmm, we need to test it in the scope of Spark and DBC.

Reply:
It should be okay because this is the version that S3N is relying on in the recent versions.

But it is definitely worth testing.